daffodil-io/src/main/scala/org/apache/daffodil/processors/charset/CharsetCompiler.scala

```diff
@@ -0,0 +1,35 @@
+
```

Comment:
Please rename this file from 'CharsetCompiler.scala' to 'BitsCharsetDefinition.scala' and remove the blank first line.